doc/manual.md

```diff
@@ -410,6 +410,49 @@ return m
 
 But we do not use this pattern in Pallene because it is harder to optimize function calls if they can be monkey-patched.
 
+## Pallene to Lua translator
```

Comment:
In this part we don't need to repeat the explanation of what Pallene is. We also don't need to explain how it is implemented -- the manual is primarily intended for end users.